bentoml/_internal/runner/runner.py

```diff
@@ -51,18 +50,13 @@ async def async_run(self, *args: t.Any, **kwargs: t.Any) -> t.Any:
         )
 
 
-# TODO: Move these to the default configuration file and allow user override
-GLOBAL_DEFAULT_MAX_BATCH_SIZE = 100
-GLOBAL_DEFAULT_MAX_LATENCY_MS = 10000
-
-
 @attr.define(slots=False, frozen=True, eq=False)
 class Runner:
     runnable_class: t.Type[Runnable]
     runnable_init_params: dict[str, t.Any]
     name: str
     models: list[Model]
-    resource_config: Resource
+    resource_config: dict[str, t.Any]
```

Comment:
nit: Config can be injected as default value through `Provide[BentoMLContainer.key_name]`

Reply:
Nevermind, I thought this was the method arguments. We can just default to the global config here, but we should just use `get`; I don't think simple_di supports `Provide` on class attributes? @bojiang